lib/modulesync/pr/github.rb

```diff
@@ -18,18 +18,22 @@ def manage(namespace, module_name, options)
         head = "#{namespace}:#{options[:branch]}"
         target_branch = options[:pr_target_branch] || 'master'
 
-        pull_requests = @api.pull_requests(repo_path, :state => 'open', :base => target_branch, :head => head)
-        if pull_requests.empty?
-          pr = @api.create_pull_request(repo_path,
-                                        target_branch,
-                                        options[:branch],
-                                        options[:pr_title],
-                                        options[:message])
-          $stdout.puts \
-            "Submitted PR '#{options[:pr_title]}' to #{repo_path} - merges #{options[:branch]} into #{target_branch}"
+        if options[:noop]
+          puts "Using no-op. Submitted PR '#{options[:pr_title]}' to #{repo_path} - merges #{options[:branch]} into #{target_branch}"
```

Comment:
I don't find this really clear. This message seems to say that even though `noop` is used, a PR is being actually submitted anyway…

Reply:
Agreed. Using `would` feels clearer.
```suggestion
          puts "Using no-op. Would submit PR '#{options[:pr_title]}' to #{repo_path} - merges #{options[:branch]} into #{target_branch}"
```

Reply:
True. I had this with "would".

The reason why I switched it back to the same wording as the actual, non-noop output is for the tests: It makes it easy to check for the same sentence.

But, really, this is a hack, true. The test should verify what is actually being tested. Currently, this is "slight cheating". I'll fix it.

Reply:
Done. Note that the tests are failing _as expected_. It demonstrates that there is an issue and the tests are able to catch it.